src/containers/IssueCard/IssueCard.js

```diff
@@ -8,7 +8,7 @@ import { LabelBadge } from '../../components/LabelBadge';
 
 import { ContextualDropdown } from './ContextualDropdown';
 
-const StatusIndicator = ({ status }) => {
+const CIStatusIndicator = ({ status }) => {
```

Comment:
These status aren't exclusive to CI if I'm not mistaken, I'd rather have it named `ChecksStatusIndicator` (this will soon use the Github Checks API once it's available on the Graphql endpoint)